pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -585,7 +654,59 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *api.Pod, _ api.PodStatus, podSt
 		return
 	}
 
-	// Step 5: start containers in podContainerChanges.ContainersToStart.
+	// Step 5: start init containers.
+	status, next, done := findNextInitContainerToRun(pod, podStatus)
```

Comment:
New function name looks fine, but I wonder why we need to do pod status again to know the next init container to start?
Shouldn't computeChanges() do the job?